cmd/modern/root/install/mssql-base.go

```diff
@@ -535,12 +585,12 @@ WHERE IsPresent = 1
 SET @sql = SUBSTRING(@sql, 1, LEN(@sql)-1)
 EXEC(@sql)`
 
-	c.query(fmt.Sprintf(text, temporaryFolder, file, fileNameWithNoExt, temporaryFolder, file))
+	c.query(fmt.Sprintf(text, temporaryFolder, file, databaseName, temporaryFolder, file))
 
 	alterDefaultDb := fmt.Sprintf(
 		"ALTER LOGIN [%s] WITH DEFAULT_DATABASE = [%s]",
 		userName,
-		fileNameWithNoExt)
+		databaseName)
```

Comment:
>databaseName) [](http://example.com/codeflow?start=2&length=13)

the escaping of `'` characters for this query will be incorrect.
This query isn't injecting the name into a string for dynamic sql so only the ] should be escaped.

As I mentioned before - the identifier should be escaped only at the point of insertion, using the appropriate escaping mechanism. 
A functional test would catch this bug.